faraday-my_middleware.gemspec

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
-$LOAD_PATH.push File.expand_path('lib', __dir__)
-
-require 'faraday/my_middleware/version'
+require_relative 'lib/faraday/my_middleware/version'
```

Comment:
It's quite a common practice to expand `$LOAD_PATH` so you don't have to use `require_relative`.
I'm not against this change, but curious on why you think this is a better approach?

Reply:
> It's quite a common practice to expand `$LOAD_PATH`

I'm not a fan of "use a common practice just because it's common". 😅 (I'm also using tabs indentation even for Ruby 😶 )



> so you don't have to use `require_relative`

I don't see a problem here.



> I'm not against this change, but curious on why you think this is a better approach?

But I see a problem with `require`, and even faced with it. You know, the worst kind of errors, when there is no exception, and everything works, but slightly in another way…

So, if you have a gem installed globally (just `gem install my-gem`) — this `require` becomes ambiguous. And in the old code there is even `push`, not `unshift`, so I'm even more afraid what can happen here.

In short words: `require_relative` is explicit, reliable, doesn't require a global state change (doesn't affect anything), not too longer to write.